time stamp
uint64 command
float32 dirVecX
float32 dirVecY
float32 dirVecZ
